README.md

```diff
@@ -1,106 +1,65 @@
+- HedgeDoc 2 will be split in two components: the backend (this repo) and the frontend in
+  the [react-client repo](https://github.com/hedgedoc/react-client).
+- The 1.x release is maintenance-only. We do not accept feature requests or PRs for this release anymore and may choose
+  to close non-critical bug reports, if the bug will be non-existent in 2.0.
 
 ## Community and Contributions
 
-We welcome contributions! There's a lot to do: If you would like to report bugs,
-the [issue tracker][github-issue-tracker] is the right place. If you can help
-translating, find us on [POEditor][poeditor-url]. To get started developing,
-take a look at the [docs/dev](docs/dev) directory. In any case: come talk to us,
-we'll be delighted to help you with the first steps.
+We welcome contributions!  
+Have a look at our [contribution docs](CONTRIBUTING.md) to find out how you can help. If you want to contribute to
+HedgeDoc 2, please join our [development chat][matrix.org-dev-url].
 
 To stay up to date with our work or get support it's recommended to join our
 [Matrix channel][matrix.org-url], stop by our [community forums][codimd-community]
 or subscribe to the [release feed][github-release-feed]. We also engage in
-regular [community calls][codimd-community-calls] ([RSS](https://community.codimd.org/t/codimd-community-call/19.rss)) which you are very welcome to join.
-
-
-## Installation / Upgrading
-
-You can run CodiMD in a number of ways, and we created setup instructions for
-all of these:
-
-* [Docker](docs/setup/docker.md)
-* [Kubernetes](docs/setup/kubernetes.md)
-* [Cloudron](docs/setup/cloudron.md)
-* [LinuxServer.io (multi-arch docker)](docs/setup/docker-linuxserver.md)
-* [Heroku](docs/setup/heroku.md)
-* [Manual setup](docs/setup/manual-setup.md)
-
-If you do not wish to run your own setup, you can find a commercial offering at
-https://hackmd.io. This is not the same codebase as this one, but it is a very
-similar project.
+regular [community calls][codimd-community-calls] ([RSS](https://community.codimd.org/t/codimd-community-call/19.rss))
+which you are very welcome to join.
 
+# License
 
-## Configuration
+Licensed under AGPLv3. For our list of contributors, see [AUTHORS](AUTHORS).
 
-Theres two main ways to configure your CodiMD instance:
-[Config file](docs/configuration-config-file.md) or
-[environment variables](docs/configuration-env-vars.md). You can choose what
-works best for you.
+[matrix.org-image]: https://img.shields.io/matrix/hedgedoc:matrix.org?logo=matrix&server_fqdn=matrix.org
 
-CodiMD can integrate with
+[matrix.org-url]: https://chat.hedgedoc.org
 
-* facebook, twitter, github, gitlab, dropbox, google, ldap, saml and [oauth2](docs/guides/auth/oauth.md) **for login**
-* imgur, s3, minio, azure **for image/attachment storage** (files can also be local!)
-* dropbox **for export and import**
+[matrix.org-dev-url]: https://matrix.to/#/#hedgedoc-dev:matrix.org?via=shivering-isles.com&via=matrix.org&via=kif.rocks
```

Comment:
For later changes (and a shorter URL) just use: https://chat.hedgedoc.org/dev

Reply:
Didn't know that exists, thanks xD